src/KeyworkerProfile/container.js

```diff
@@ -45,6 +63,32 @@ class KeyworkerProfileContainer extends Component {
     return response.data;
   }
 
+  async getKeyworkerAllocations (agencyId, staffId) {
+    const response = await axiosWrapper.get('/keyworkerAllocations', {
+      headers: {
+        jwt: this.props.jwt
+      },
+      params: {
+        agencyId: agencyId,
+        keyworkerStaffId: staffId
+      }
+    });
+    return response.data;
+  }
+
+  async getKeyworkerDetails (agencyId, staffId) {
+    const response = await axiosWrapper.get('/keyworker', {
+      headers: {
+        jwt: this.props.jwt
+      },
+      params: {
+        agencyId: agencyId,
```

Comment:
Atm we dont need the agency id

Reply:
Also this is the same api call as getting the KW name and no of allocations. But I tihink this is ok for now until we get the reqs of what KW details are actually needed